microservice/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM docker.io/kennethreitz/httpbin
```

Comment:
can't it be a more basic image, like alpine? To my understanding, you only use the container to run `curl`...